internal/store/pod.go

```diff
@@ -90,9 +90,99 @@ func podMetricFamilies(allowAnnotationsList, allowLabelsList []string) []generat
 		createPodStatusScheduledFamilyGenerator(),
 		createPodStatusScheduledTimeFamilyGenerator(),
 		createPodStatusUnschedulableFamilyGenerator(),
+		createPodReadinessProbeFamilyGenerator(),
+		createPodLivenessProbeFamilyGenerator(),
+		createPodPrivilegedFamilyGenerator(),
```

Comment:
could you sort these? It would be great to also sort the definition to keep this file easier to navigate.